packages/teams-js/src/public/runtime.ts

```diff
@@ -185,6 +186,7 @@ export const teamsRuntimeConfig: Runtime = {
       },
       update: {},
     },
+    interactive: {},
```

Comment:
This probably shouldn't be here. You only need this entry here if LiveShare needs to work on an instance of Teams that is not yet using the host-sdk. Is that a scenario that needs to work for you?

Another way to think of this: putting there here means that LiveShare will show up as working on every old version of Teams that isn't using the host-sdk, whether or not it actually is. If you do need this to work on some older teams clients, you may need to make some changes to only add interactive to this layer in certain cases